SlidingWindow_cpp.ipynb

```diff
@@ -0,0 +1,93 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "SlidingWindow.cpp",
+      "provenance": [],
+      "collapsed_sections": [],
+      "authorship_tag": "ABX9TyNjt0CUM+rvhDsN9BmhWY++",
+      "include_colab_link": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    },
+    "language_info": {
+      "name": "python"
+    },
+    "accelerator": "GPU"
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/areenoverclouds/winter-of-contributing/blob/Competitive_Programming/SlidingWindow_cpp.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZsZ5UUMT56to"
+      },
+      "source": [
+        "**Sliding Window Maximum (Maximum of all subarrays of size k)**\n",
+        "\n",
+        "Language : C++\n",
+        "\n",
+        "Time Complexity : O(n*k)\n",
+        "\n",
+        "Space Complexity : O(1)\n",
+        "\n",
```

Comment:
Specify Complexity with a Title at the bottom

Reply:
Sir I have resolved the issues and opened a new pull request #3614
Sorry for the inconvenience.

Reply:
you can resolve and update this PR also. no need of creating a new PR.